misk-action-scopes/src/main/kotlin/misk/scope/ActionScopedProviderModule.kt

```diff
@@ -27,12 +28,13 @@ abstract class ActionScopedProviderModule : KAbstractModule() {
   abstract fun configureProviders()
 
   /** Binds an [ActionScoped] which only pulls from data seeded at the scope entry */
-  fun <T : Any> bindSeedData(kclass: KClass<T>) {
+  fun <T : Any> bindSeedData(kclass: KClass<T>, newThing: String?) {
     bindSeedData(Key.get(kclass.java), Key.get(actionScopedType(kclass)))
   }
 
   /** Binds an [ActionScoped] which only pulls from data seeded at the scope entry */
-  fun <T : Any> bindSeedData(type: TypeLiteral<T>) {
+  @JvmOverloads
+  fun <T : Any> bindSeedData(type: TypeLiteral<T>, newThing: String? = null) {
```

Comment:
The combination of `@JvmOverloads` and default value for make this a on-breaking change, and revapi doesn't fail on this.